src/protocol/kademlia/impl/validator_default.cpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <libp2p/protocol/kademlia/impl/validator_default.hpp>
+
+#include <libp2p/protocol/kademlia/error.hpp>
+
+namespace libp2p::protocol::kademlia {
+
+  outcome::result<void> ValidatorDefault::validate(const Key &key,
+                                                   const Value &value) {
+    return outcome::success();
```

Comment:
At least we can validate the case when a peer's public key is stored.

```
key is "/pk/<peerid>"
value is the public key
```

The simplest validation could be ```SHA256(value) == peerid``` 
https://github.com/libp2p/specs/tree/cb971aa17b2ee6bb7f20df6daed2e7cd4c7e861d/kad-dht

Reply:
It should provide by external implementations of specified validator.